packages/sources/ncfx/src/transport/crypto.ts

```diff
@@ -76,6 +76,9 @@ export const transport = new WebSocketTransport<WsTransportTypes>({
         return
       }
 
+      const providerTime = message.timestamp.includes('Z')
+        ? message.timestamp
+        : `${message.timestamp}Z`
```

Comment:
Could we add a comment explaining that if we don't have a timezone we're assuming the timestamp is in UTC?

Would also be helpful to include an example timestamp in the comment, I originally though this was an epoch timestamp (e.g. `1694168955993`) which confused me.

Reply:
good point, added.